src/debug/daccess/dacdbiimplstackwalk.cpp

```diff
@@ -1152,7 +1152,7 @@ CorDebugInternalFrameType DacDbiInterfaceImpl::GetInternalFrameType(Frame * pFra
 void DacDbiInterfaceImpl::UpdateContextFromRegDisp(REGDISPLAY * pRegDisp,
                                                    T_CONTEXT *  pContext)
 {
-#if defined(_TARGET_X86_)
+#ifndef WIN64EXCEPTIONS
```

Comment:
I'd prefer using `#if defined(_TARGET_X86_) && !defined(WIN64EXCEPTIONS)` here. While these are equivalent, it better expresses what the code below is for.